README.md

```diff
@@ -139,6 +139,39 @@ only really makes sense if you're calling Grover directly (and not via middlewar
 Grover.new('<some URI with basic authentication', username: 'the username', password: 'super secret').to_pdf
 ```
 
+#### Adding cookies
+If you need to set cookies to your new page instance, just pass them along like this:
+
+```ruby
+myCookies = [{:name=>"sign_username", :value=>"any@any.com", :domain=>"mydomain"}, {:name=>"_session_id", :value=>"9c014df0b699d8dc08d1c472f8cc594c", :domain=>"mydomain"}]
```

Comment:
prefer newer hash notation rather than rockets:

```suggestion
myCookies = [
  { name: 'sign_username', value: 'any@any.com', domain: 'mydomain' },
  { name: '_session_id', value: '9c014df0b699d8dc08d1c472f8cc594c', domain: 'mydomain' }
]
```